doc/api/console.markdown

```diff
@@ -89,36 +124,53 @@ Defaults to `false`. Colors are customizable; see
 
 ### console.error([data][, ...])
 
-Same as [`console.log()`][] but prints to stderr.
+Prints to stderr with newline. Multiple arguments can be passed, with the first
+used as the primary message and all additional used either as substitution
```

Comment:
is "either" meant to be in here? Sounds a little confusing without having an or here